.github/workflows/check_unit_tests.yml

```diff
@@ -34,22 +36,47 @@ jobs:
         sudo apt-get update && sudo apt-get install -y ccache
         bash ./buildscripts/ci/tools/setup_ccache_config.sh
 
+    - name: Free up disk space
+      run: |
+        sudo docker system prune -a -f
+        sudo rm -rf /usr/local/lib/android
+
     - name: Setup environment
       run: |
         bash ./buildscripts/ci/linux/setup.sh
     - name: Build
       run: |
         mkdir -p build.artifacts/env
-        bash ./buildscripts/ci/linux/build_utest.sh -n ${{ env.BUILD_NUMBER }}
+
+        ENABLE_CODE_COVERAGE='false'
+
+        if [[ "${{ github.event_name }}" == "schedule" ]]; then
+          ENABLE_CODE_COVERAGE='true'
+        else
+          ENABLE_CODE_COVERAGE='false'
+        fi
```

Comment:
it would be great to be able to run manually with the ability to enable code coverage